website/blog/2022/02/10/splunk-apisix-integration.md

```diff
@@ -37,7 +37,7 @@ This article explains how to configure and use the [Splunk HEC](https://docs.spl
 
 [Splunk HTTP Event Collector (HEC)](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector)is an HTTP event collector provided by Splunk that provides the ability to send data and application events to Splunk using the HTTP(S) protocol.
 
-## About splunk-hec-logging lugin
+## About splunk-hec-logging plugin
 
 The [splunk-hec-logging](https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/splunk-hec-logging.md) is used to forward Apache APISIX request logs to Splunk for analysis and storage. When enabled, Apache APISIX will take the request context information during the Log phase, serialize it into [Splunk Event Data Format](https://docs.splunk.com/Documentation/Splunk/latest/Data/FormateventsforHTTPEventCollector#Event_metadata) and submit it to the batch queue. The data in the queue is committed to Splunk HEC when the maximum processing capacity of the batch queue per batch is triggered, or when the maximum time to refresh the buffer is reached.
```

Comment:
1. What does it mean by saying "take the request context information during the Log phase"? 
My guess is that "Apache APISIX will collect the request context information while tracking or logging the request"?

2. Put the condition or trigger before the thing that follows to make the context more logic and coherent.

```suggestion
When the maximum processing capacity of a queue is reached or when the maximum time to refresh the buffer is reached, data in the queue will be committed to Splunk HEC.
```

Reply:
Fixed.